src/services/content.service.ts

```
@@ -594,14 +615,16 @@ export class contentService {
       }
 
       // Remove Ortho complexity
-      if (contentData.length < limit && contentType.toLocaleLowerCase() !== 'paragraph') {
-
+      if (
+        contentData.length < limit &&
+        contentType.toLocaleLowerCase() !== 'paragraph'
+      ) {
         mileStoneQuery = mileStoneQuery.filter((mileStoneQueryEle) => {
           return !mileStoneQueryEle.hasOwnProperty('totalOrthoComplexity');
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)